apps/publish.sh

```diff
@@ -41,6 +41,8 @@ PLATFORMS=${MANIFEST_PLATFORMS_DEFAULT-""}
 require_params FACTORY ARCHIVE TARGET_TAG
 #-- END: Input params
 
+apk add tar
```

Comment:
the way i call tar requires the gnu version of tar and not busybox-tar. We can update lmp-image-tools if this PR is okay. but i have a feeling you'll want to completely overhaul this approach, so i left it like this to keep it testable in my factory.